uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/api/EventFilter.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.mqtt.client.control.api;
+
+import lombok.Getter;
+import lombok.NonNull;
+
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Filter of events in event storage.
+ */
+@Getter
+public final class EventFilter {
+    private final Event.Type type;
+    private final String connectionName;
+    private final String topic;
+    private final String topicFilter;
+    private final byte[] content;
+
+    EventFilter(Builder builder) {
+        super();
+        this.type = builder.type;
+        this.connectionName = builder.connectionName;
+        this.topic = builder.topic;
+        this.topicFilter = builder.topicFilter;
+        this.content = builder.content;
+    }
+
+    /**
+     * Builder of event filter.
+     */
+    public static class Builder {
```

Comment:
Could we simply use @Builder annotation from lombok ? https://projectlombok.org/features/Builder

Reply:
lombok Build is not sutiable in that case due to we have overloaded withContent() method with String and byte[] arguments. I checked lombok manual and did not found ability to add extra methods to it builder.